recipes-rubygems/rubygems-ffi_1.15.1.bb

```diff
@@ -6,11 +6,11 @@ HOMEPAGE = "https://github.com/ffi/ffi/wiki"
 LICENSE = "BSD-3-Clause"
 LIC_FILES_CHKSUM = "file://COPYING;md5=0b4e3af2fd54eb00a94b967e6b927818"
 
-EXTRA_RDEPENDS += "\
+EXTRA_RDEPENDS_append += " \
     libffi \
 "
 
-GEM_INSTALL_FLAGS += "\
```

Comment:
potential issue: Use '_append' instead of ' += ' as it overwrites 'EXTRA_RDEPENDS ?= ""